client/my-sites/domains/domain-management/transfer/transfer-to-other-user/transfer-domain-to-other-user.jsx

```diff
@@ -225,7 +225,7 @@ class TransferDomainToOtherUser extends Component {
 
 		if ( isMapping ) {
 			return translate(
-				'Do you want to transfer the domain mapping of {{strong}}%(domainName)s{{/strong}} ' +
+				'Do you want to transfer the domain connection of {{strong}}%(domainName)s{{/strong}} ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Do you want to transfer the domain mapping of {{strong}}%(domainName)s{{/strong}} to {{strong}}%(selectedUserDisplay)s{{/strong}}?' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domain-transfer-dialog-text)